include/mqt-core/dd/Simulation.hpp

```diff
@@ -42,13 +43,17 @@ namespace dd {
  * mid-circuit measurements, see @ref sample(const QuantumComputation&,
  * std::size_t, std::size_t).
  *
+ * @tparam stgy The approximation strategy applied.
  * @param qc The quantum computation to simulate
  * @param in The input state to simulate. Represented as a vector DD.
  * @param dd The DD package to use for the simulation
+ * @param approx The object describing the approximation strategy.
  * @return A vector DD representing the output state of the simulation
  */
+template <const ApproximationStrategy stgy = None>
```

Comment:
```suggestion
template <const ApproximationStrategy strategy = None>
```
No need to be cryptic with the names here in general. I'd much prefer to spell these out.